bq_schemas/bigquery_views_legacy_sql/datasets_cost_estimation_v1_0.sql

```diff
@@ -0,0 +1,15 @@
+-- This view provides BigQuery datasets current estimated costs, which might differ to those provided by GCP billing
```

Comment:
Why do we keep legacy views? Is it possible switch all to standard? And if so how much is it to work?
I believe We should forget about legacy sql ASAP as it's gonna be harder later while develop of new features.

Reply:
We initially used legacy views and we want to keep it consistent, so that user can choose the preferred option.